pkg/internalapis/cnsoperator/register.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// NOTE: Boilerplate only.  Ignore this file.
+
+// Package apis v1alpha1 contains API Schema definitions for the cns v1alpha1 API group
+// +k8s:deepcopy-gen=package,register
+// +groupName=cns.vmware.com
+package apis
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cnsfilevolumeclientv1alpha1 "sigs.k8s.io/vsphere-csi-driver/pkg/internalapis/cnsoperator/cnsfilevolumeclient/v1alpha1"
+	triggercsifullsyncv1alpha1 "sigs.k8s.io/vsphere-csi-driver/pkg/internalapis/cnsoperator/triggercsifullsync/v1alpha1"
+)
+
+// GroupName represents the group for cns operator apis
+const GroupName = "cns.vmware.com"
+
+// Version represents the version for cns operator apis
+const Version = "v1alpha1"
+
+var (
+	// SchemeGroupVersion is group version used to register these objects
+	SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: Version}
+
+	// CnsFileVolumeClientPlural is plural of CnsFileVolumeClient
+	CnsFileVolumeClientPlural = "cnsfilevolumeclients"
```

Comment:
Can you clarify why are we touching existing APIs?

Reply:
same as above.